.travis/test.sh

```diff
@@ -104,6 +106,7 @@ The backup database file is sandbox/${db}2.db
 
 ovn_start_db nb standalone 1 $srcdir/ovn/ovn-nb.ovsschema
 
-cd ../goovn; go test
+export GO111MODULE=on
```

Comment:
Please keep the go mod related changes in the go mod patch, instead of this one.